controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -145,7 +145,11 @@ def run_match(supervisor: Supervisor) -> None:
         robot.getField('customData').setSFString('start')
 
     # ... then un-pause the simulation, so they all start together
-    supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_RUN)
+    if supervisor.getDevice("2021a-compatibility") is None:
+        # webots-2021a removed the RUN mode and now uses FAST
```

Comment:
Given this comment, are these the right way around?

Reply:
More likely the comment is unclear. If 2021a-compatibility is None, you are running 2020b

Reply:
The comments are changed in 8873fdc54fb38a5f3fb498eed09181bc475a72af